src/app/components/endpoint-list/endpoint-details/endpoint-details.component.ts

```diff
@@ -74,11 +70,28 @@ export class EndpointDetailsComponent implements OnInit {
 
   toggleStatusFilter(option) {
     this.statusFilter[option] = !this.statusFilter[option]
+    this.setFilterToLStorage();
   }
 
   selectNamespace(event, namespace) {
     event.preventDefault();
     this.namespace = namespace;
   }
 
+  // private
+
+  private setFilterOptions() {
+    let options = JSON.parse(localStorage.getItem('jobsStatusFilter'));
+    if (options) {
+      this.statusFilter = options;
+    } else {
+      this.statusFilter = { success: true, running: true, failed: false };
+      this.setFilterToLStorage()
```

Comment:
As for me I was confused when I read this method name setFilterToLStorage. Is'n it better to name it as setFilterToLocalStorage ?